pkg/agent/ngt/service/ngt.go

```diff
@@ -119,7 +122,7 @@ func New(cfg *config.NGT) (nn NGT, err error) {
 			return err
 		}))
 		eg.Go(safety.RecoverFunc(func() (err error) {
-			if len(n.path) != 0 {
+			if len(n.path) != 0 && !n.inMem {
```

Comment:
![badge](https://img.shields.io/badge/review-note-informational.svg?style=flat-square)
This change may not affect because the value of `n.inMem` is checked at L116.